## Практическая работа №2 Теория вероятностей и математическая статистика

#### Задание №1
##### Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.

Для решения задачи воспользуемся формулой бернули
$$P(X=k) = C^n_{k}p^kq^{n-k}$$
$$p = 0.8; q = 1 - p = 0.2$$
$$P(X=85) = C^{85}_{100}p^{85}q^{15} = \frac{100!}{85!(100-85)!}*0.8^{85}*0.2^{15}$$
с такими числами проще написать python функцию для расчётов

In [22]:
import numpy as np
def bern(n:int, k:int, p:float) -> float:
    """
    Реализация формулы Бернули.
    :param n: Количество независящих испытаний
    :param k: Количество наступления события А
    :param p: Вероятность наступления события А
    :return: Вероятность наступления события А k раз в n испытаниях
    """
    return (np.math.factorial(n)/(np.math.factorial(k)*np.math.factorial(n-k)))*(p**k)*(1-p)**(n-k)

In [23]:
P = bern(100, 85, 0.8)
print('P(k=85) = %.4f' % P)

P(k=85) = 0.0481


##### Ответ: Вероятность того что стрелок попадёт в мишень ровно 85 раз развна 0.0481 (4.81\%)

#### Задание №2
##### Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек. Какова вероятность, что ни одна из них не перегорит в первый день? Какова вероятность, что перегорят ровно две?

Применим формулу Пуассона, зная, что m = 0 и $\lambda = np = 5000*0.0004 = 2$
$$P_{0} = \frac{\lambda^m}{m!}e^{-\lambda} = \frac{2^0}{0!}e^{-2} = 1*e^{-2} = 0.1353$$

##### Значит, вероятность того что перегорит одна лампочка в первый день равна 0.1353 (13.53%)

Найдём вероятность того что перегорят ровно 2 лампочки. m = 2
$$P_{2} = \frac{\lambda^m}{m!}e^{-\lambda} = \frac{2^2}{2!}e^{-2} = 2*e^{-2} = 0.2706 = 27.06\%$$
##### Ответ:
##### Вероятность того что ни одна ламвочка не перегорит в первый день равна 0.1353 (13.53\%).
##### Вероятность того что перегорит ровно 2 лампочки равна 0.2706 (27.06\%)

#### Задание №3
##### Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

#### Задание №4
##### В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. Какова вероятность того, что все мячи белые? Какова вероятность того, что ровно два мяча белые? Какова вероятность того, что хотя бы один мяч белый?